src/index.ts

```diff
@@ -315,3 +328,18 @@ export class ProjectConfig {
   }
 
 }
+
+// Gets the json schema for polymer.json, generated from the typescript
+// interface for runtime validation. See the build script in package.json for
+// more info.
+const getSchema: () => jsonschema.Schema = (() => {
```

Comment:
this is a little complex. Why not just:

```typescript
let schema: jsonschema.Schema | undefined = undefined;
function getSchema() {
  if (schema === undefined) {
    schema = JSON.parse(
        fs.readFileSync(path.join(__dirname, 'schema.json'), 'utf-8'));
  }
  return schema;
}
```

you're only hiding the variable from ourselves with the closure.